testing/fixture_test.h

```diff
@@ -19,8 +19,14 @@ namespace testing {
 
 class FixtureTest : public ThreadTest {
  public:
+  // Uses the default filenames from the fixtures generator.
   FixtureTest();
 
+  // Allows to customize the kernel, ELF and split ELF filenames.
+  explicit FixtureTest(std::string kernel_filename,
```

Comment:
`explicit` is no longer needed since there is more than one argument.

Reply:
Done